backend/Services/SpeciesServices.cs

```diff
@@ -0,0 +1,24 @@
+using WhaleSpotting.Models.Data;
+using WhaleSpotting.Models.Request;
+
+namespace WhaleSpotting.Services;
+
+public interface ISpeciesServices
```

Comment:
As above for convention this should be `ISpeciesService`

Reply:
Change made